backend/src/database/migrations/20221116075840-update-get-next-job.js

```diff
@@ -0,0 +1,94 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createFunction(
+      'get_next_job',
+      [{ type: 'text', name: 'worker' }],
+      'json',
+      'plpgsql',
+      `
+      UPDATE job_queue
+      SET worker_id = worker, status = 'ENQUEUED', updated_at = clock_timestamp()
+      WHERE
+      -- worker is not already operating on a job
+      NOT EXISTS (SELECT 1 FROM job_queue q WHERE q.worker_id = worker LIMIT 1)
+      AND id = ( SELECT q.id
+          FROM job_queue q, campaigns p, credentials c
+          WHERE q.status = 'READY'
+          AND q.campaign_id = p.id
+          AND p.cred_name = c.name
+          AND is_credential_used(p.id) IS NULL
+          AND q.visible_at <= clock_timestamp()
```

Comment:
this line is the key change right? would be good to have comments summarising the diff so it's easier to follow 😅

Reply:
done